modules/install-secured-cluster-operator.adoc

```diff
@@ -16,7 +16,7 @@ You can install secured cluster services on your clusters by using the `SecuredC
 
 [CAUTION]
 ====
-To install Collector on systems that have Unified Extensible Firmware Interface (UEFI) and that have Secure Boot enabled, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages. Collector identifies Secure Boot status at the start and switches to eBPF probes if required.
+Collector is installed when installing secured cluster services. To install Collector on systems that have Unified Extensible Firmware Interface (UEFI) and that have Secure Boot enabled, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages. Collector identifies Secure Boot status at the start and switches to eBPF probes if required.
```

Comment:
"installed when installing" tripped me up a bit. What if you said something like:
```suggestion
When installing secured cluster services, Collector is also installed. To install Collector on systems that have Unified Extensible Firmware Interface (UEFI) and that have Secure Boot enabled, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages. Collector identifies Secure Boot status at the start and switches to eBPF probes if required.
```
Not sure if that is better. I am sure you could come up with something that fits better here.